nebula/ui/components/VPNServerList.qml

```diff
@@ -167,8 +167,10 @@ FocusScope {
                             horizontalAlignment: Text.AlignLeft
                             // TODO: Replace placeholder strings and generate
                             // values that will be set instead of `%1`
-                            text: !statusComponent.rowShouldBeDisabled
-                                ? "Last updated %1 ago."
+                            text: VPNServerLatency.isActive
+                                ? "Checking... %1%".arg(Math.round(VPNServerLatency.progress * 100))
+                                : (VPNController.state === VPNController.StateOff)
+                                ? "Last updated %1 ago.".arg(VPNServerLatency.lastUpdateTime)
```

Comment:
At least on MacOS due to the date format coming back, having "ago" at the end of the string doesn't seem right.

![Screenshot 2023-02-09 at 10 00 31 AM](https://user-images.githubusercontent.com/3746552/217900607-075b04f0-4dea-4c4c-a79c-a230fef1699c.png)